src/clients/users_plugin_ldap_shadowaccount.rb

```diff
@@ -121,17 +123,9 @@ def main
           @last_change = _("Never")
         end
         if @expires != "0" && @expires != "-1" && @expires != ""
-          @out = Convert.to_map(
-            SCR.Execute(
-              path(".target.bash_output"),
-              Ops.add(
-                Builtins.sformat(
-                  "date --date='1970-01-01 00:00:01 %1 days' ",
-                  @expires
-                ),
-                "+\"%Y-%m-%d\""
-              )
-            )
+          @out = SCR.Execute(
+            path(".target.bash_output"),
+            "/usr/bin/date --date='1970-01-01 00:00:01 %1 days' #{@expires.to_s.shellescape}+\"%Y-%m-%d\""
```

Comment:
This interpolation looks wrong. Should not be `%1` replaced with `@expires.to_s.shellescape`=